app/setup_handlers.go

```diff
@@ -50,8 +49,8 @@ func SetupHandlers(app *App) {
 		}
 	}
 	baseAppLegacySS := app.ParamsKeeper.Subspace(baseapp.Paramspace).WithKeyTable(paramstypes.ConsensusParamsKeyTable())
-	app.UpgradeKeeper.SetUpgradeHandler(releaseVersion, func(ctx sdk.Context, plan types.Plan, vm module.VersionMap) (module.VersionMap, error) {
-		app.Logger().Info("Running upgrade handler for " + releaseVersion)
+	app.UpgradeKeeper.SetUpgradeHandler(constant.Version, func(ctx sdk.Context, plan types.Plan, vm module.VersionMap) (module.VersionMap, error) {
```

Comment:
Right now this value is empty

Reply:
It's empty in the code. It is set by the linker:

https://github.com/zeta-chain/node/blob/1ed1afcffc0299a3994db5359bcfb6e56b565d8e/Makefile#L16